src/test/java/com/cloudbees/jenkins/plugins/sshagent/SSHAgentStepWorkflowTest.java

```diff
@@ -282,4 +283,24 @@ public void sshAgentDocker() throws Exception {
         });
     }
 
+    @Issue("JENKINS-32104")
+    @Test
+    public void testMissingCredential() {
+        assumeFalse(Functions.isWindows());
+
```

Comment:
```suggestion
```
Does not seem necessary in this case, since the test does not rely on Linux specifics like `sh`.